trade-finance-manager-ui/server/controllers/case/cancellation/reason-for-cancelling.controller.post.test.ts

```diff
@@ -142,6 +142,38 @@ describe('postReasonForCancelling', () => {
           dealId,
           errors: validationErrors,
           reasonForCancelling,
+          previousPage: `/case/${dealId}/deal`,
+        });
+      });
+
+      it('renders the page with the back URL as the check details page when "change" is passed in as a query parameter', async () => {
```

Comment:
Seems that we don't need this test, can just update the test above to consume `getPreviousPageUrlForCancellationFlow` in `previousPage` 😎

Reply:
See above, hoping to leave this as is🤞

Reply:
As per previous comment thread, resolving here (and open to further discussion outside the PR)